.eslintrc

```diff
@@ -13,6 +13,9 @@
     "no-inline-comments": 0,
     "camelcase": 0,
     "react/forbid-prop-types": 0
+  },
+  "parserOptions": {
```

Comment:
Does this "allow" ES7 syntax for all code? Or does it just provide the ability to parse such code without crashing? 

If it allows it for non-test code, it may cause some issues since our babel config doesn't support it. 

Maybe better to make a separate .eslintrc file in the node test directory for now.

Reply:
It is used to remove lint error when I use ES7 syntax on test code, I think it will take effects on non-test code. Thought it's OK, will change it as you suggested.